### PR TITLE
Create sprint-94-summary.md

### PR DESCRIPTION
**Pull request by reitermb**
## Summary of Changes
Adds sprint 94 summary
Resolves final fix for Sprint 92/93 to reflect when #2820 closed

### CODE REVIEW

**Review by andrew-jameson — APPROVED** [review]

File: docs/Sprint-Review/sprint-94-summary.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+
+
+
+### Submitted (QASP Review, OCIO Review)
+* [#2681 TANF Section 1 Validation cleanup](https://github.com/raft-tech/TANF-app/issues/2681)
+* [#2799 Generate error mismatching field rpt_month_year w/ header](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2799)
+* [#2793 update dateYearIsLargerThan() validator logic for rpt_month_year](https://github.com/raft-tech/TANF-app/issues/2793)
+* [#1441 As tech lead I need new permissions group for OFA data analysts](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/1441)
+* [#2673 Cat 1 errors audit fixes](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2673)
```
Comment (ADPennington) [inline docs/Sprint-Review/sprint-94-summary.md]: @reitermb i think 2673 was put into qasp by mistake. still in raft review.

File: docs/Sprint-Review/sprint-94-summary.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    * Error Categories GitHub Documentation
+    * Friendly name fixes (#2801)
+    * Knowledge Center Content (#2847, #2846)
+        * Submission History
+        * Error reports / data file structure
+   
+
+
+## Tickets
+### Completed/Merged
```
Comment (ADPennington) [inline docs/Sprint-Review/sprint-94-summary.md:29]: @reitermb #2825 missing here.

File: docs/Sprint-Review/sprint-94-summary.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
+# Sprint 94 Summary
+
+02/28/2024 - 03/12/2024
+
+Velocity (Dev): 16
```
Comment (ADPennington) [inline docs/Sprint-Review/sprint-94-summary.md:5]: @reitermb please double-check velocity with changes mentioned below.

**Review by ADPennington — CHANGES_REQUESTED** [review]
please see comments.

File: docs/Sprint-Review/sprint-94-summary.md (hunk 2 above)
Comment (reitermb) [reply docs/Sprint-Review/sprint-94-summary.md:29]: Adding to Ready To Merge to reflect the state as of yesterday!

File: docs/Sprint-Review/sprint-94-summary.md (hunk 3 above)
Comment (reitermb) [reply docs/Sprint-Review/sprint-94-summary.md:5]: Velocity should still be right given that #2825 merged in today rather than yesterday!

**Comment by reitermb** [comment]
> please see comments.

Should all be addressed now!

**Review by ADPennington — APPROVED** [review]
👍🏾

**Pull request merged**